scrape/scrape.go

```diff
@@ -1573,6 +1574,7 @@ loop:
 		if ok {
 			ref = ce.ref
 			lset = ce.lset
+			hash = ce.hash
```

Comment:
can't see why this is necessary

Reply:
That’s a left over from previous version that needed hash to be always set for a lookup.

https://github.com/prometheus/prometheus/commit/3a4fcebf4facd6c5da64e1ea35ab6c74e8ce6833